project_timesheet_time_control/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa
```

Comment:
@pedrobaeza If you work full time for a company, the company is the copyright owner of your code.